### DISCUSSION TITLE
Custom error in controlled input

### DISCUSSION THREAD

**Discussion by hornta** [react-hook-form/react-hook-form#9038] [Ideas] [answered]
It would be nice for a way to tell RHF a controlled input control is failing validation so that I can't submit unless I fix the error. The only way to use the `setError` now is via the register function but I'm using the `useController`.

**Comment by Moshyfawn** [comment, marked as answer]
```jsx
<Controller
  rules={{
    required: true,
    validate: value => validateField(value)
  }}
/>
```
